# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xf86-input-sample/fix-includes.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- xf86-input-sample-1.0.0/src/sample.c.orig	2006-07-13 23:31:28.000000000 -0700
+++ xf86-input-sample-1.0.0/src/sample.c	2006-07-13 23:32:40.000000000 -0700
@@ -41,12 +41,12 @@
  ****************************************************************************/
 
 #include <misc.h>
+#include <xisb.h>
 #include <xf86.h>
-#define NEED_XF86_TYPES
+// #define NEED_XF86_TYPES
 #include <xf86_ansic.h>
 #include <xf86_OSproc.h>
 #include <xf86Xinput.h>
-#include <xisb.h>
 #include <exevents.h>			/* Needed for InitValuator/Proximity stuff	*/
 
 /*****************************************************************************
